vscode-lean4/package.json

```
@@ -278,6 +278,12 @@
 				"title": "Infoview: Toggle Expected Type",
 				"description": "Toggle the display of expected type."
 			},
+			{
+				"command": "lean4.infoview.goToDefinition",
+				"category": "Lean 4",
+				"title": "Go to Definition",
+				"description": "Go to definition for the currently selected element in the InfoView."
```

```suggestion
				"title": "(Internal) Go to Definition",
				"description": "This command is an implementation detail of the 'Go to Definition' context menu option in the infoview."
```
I would make it super explicit that this is a impl detail, and one should look at the infoview code to understand what's actually happening.

---

The context menu displays the title. I believe the description is not user-facing, though.

---

I missed that, let's just change `description` then.